### PR TITLE
team tanay challenege submission

### PR DESCRIPTION
**Pull request by shivambhatia5432**

### CODE REVIEW

File: content/participants/ttjc@shivambhatia5432/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### how to contact me?
+
+- [LinkedIn](https://www.linkedin.com/in/shivam-bhatia-a83ab4185/)
+
+### projects
+It is a simple Google Keep clone- https://notes-keeper-app-shivam.netlify.app <br />
+It is a simple weather website that updates background with city images- https://weather-app-shivam.netlify.app <br />
+It is a simple recipe app- https://recipe-app-shivam.netlify.app <br />
+These 3 projects are based on React only and are quite basic because I am new to React but I have many other projects based on HTML/CSS,Javascript,Jquery,MongoDB,SQL,Express,Node.js and EJS that you can see on my Github.
+
```
Comment (gkanishk) [inline content/participants/ttjc@shivambhatia5432/index.md:42]: Please add a proper description for each project:
_description_:
_hosted link_:
_github link_:

File: content/participants/ttjc@shivambhatia5432/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### blogs
+
+1) https://medium.com/@shivambhatia_64475/top-5-services-for-developers-to-show-their-web-projects-for-free-b51b1ad0c9e9 <br />
+2) https://medium.com/@shivambhatia_64475/top-free-apis-for-web-development-beginners-e0cb23883ce <br />
+3) https://medium.com/@shivambhatia_64475/why-new-developers-should-learn-full-stack-development-first-and-then-go-for-specialisation-e0b44a1c391f <br />
```
Comment (gkanishk) [inline content/participants/ttjc@shivambhatia5432/index.md:47]: Same goes for blogs add description and link:

**Comment by shivambhatia5432** [comment]
Thank you for suggesting the changes do check and give feedbak :)

**Comment by gkanishk** [comment]
Mention hosted link also

**Comment by shivambhatia5432** [comment]
I did commit all the required changes.
My projects are in a repository I did mention their github link at top.
Then there is description of each project as these are simple projects so they can be explained in one line.
And each project has the link to open them.

**Pull request closed without merging**